### PR TITLE
changeling biomass adjustments for rotting corpses + salvage corpse adjustments

### PR DESCRIPTION
**Pull request by Darkmajia**
balance adjustments for changelings when it comes to rotting corpses. once the "extremely bloated" stage of rotting is reached, changelings are no longer capable of absorbing a corpse. unidentified corpses (mostly found through salvage) spawn in the extremely bloated stage of rotting. corpses in other stages of rotting grant half the biomass of a regular corpse, and grant no evolution points

inspect text has been added to reflect these changes that _only_ shows up for changelings, and doesn't show if the corpse has already been absorbed

image
image
image

**Changelog**
:cl:
- tweak: Salvage corpses spawn heavily rotted, and extremely bloated.
- tweak: Changelings are incapable of absorbing corpses once they reach the "extremely bloated" stage.

### CODE REVIEW

**Comment by Darkmajia** [comment]
this partially resolves #856, heretics havent been addressed but i will need to think a little bit harder on it. this also doesn't punish changelings for using corpses below the "extremely bloated" rotting threshold, though i'm open to changing it if anyone would like to give feedback before this potentially gets merged (maybe just make it any amount of rotting on a corpse? i dunno)

**Comment by Darkmajia** [comment]
added a few things to this pr, feel free to give feedback or hit that like and merge button

**Review by starlighthowls — APPROVED** [review]

**Pull request merged**
